src/open_company_web/components/create_update.cljs

```diff
@@ -0,0 +1,179 @@
+(ns open-company-web.components.create-update
+  (:require [om.core :as om :include-macros true]
+            [om-tools.core :as om-core :refer-macros (defcomponent)]
+            [om-tools.dom :as dom :include-macros true]
+            [dommy.core :refer-macros (sel)]
+            [goog.object :as gobj]
+            [goog.events :as events]
+            [goog.events.EventType :as EventType]
+            [open-company-web.api :as api]
+            [open-company-web.urls :as oc-urls]
+            [open-company-web.router :as router]
+            [open-company-web.dispatcher :as dis]
+            [open-company-web.lib.utils :as utils]
+            [open-company-web.lib.responsive :as responsive]
+            [open-company-web.components.ui.footer :refer (footer)]
+            [open-company-web.components.ui.navbar :refer (navbar)]
+            [open-company-web.components.topics-columns :refer (topics-columns)]
+            [open-company-web.components.su-preview-dialog :refer (su-preview-dialog)]
+            [clojure.data :as cd]))
+
+(defn ordered-topics-list []
+  (let [topics (sel [:div.create-update-topics-list :div.oc-active])
+        topics-list (for [topic topics] (.data (js/jQuery topic) "topic"))]
+    (vec (remove nil? topics-list))))
+
+(defn remove-pinned [data]
```

Comment:
I don't love the construction of this at all.

That's a lot of code for no comment(s) and/or docstring.

Use of `loop`, `recur` is a bit of a code smell to me. I've found there is almost always a better, more functional construction.

Lastly, this seems prone to an infinite loop if any of this data/logic goes wrong.

Strongly suggest we refactor this.